back-end/api/controllers/transcript_controller.js

```diff
@@ -34,9 +34,11 @@ export default class TranscriptController {
         try {
             const body = JSON.stringify(req.body.payload);
             const transcript = JSON.parse(body);
-            await this.#inputBoundary.formatTranscript(transcript);
+            console.log(transcript)
```

Comment:
I don't think we should have console.logs in the final version of our codes. I think I also left one in the interactor which I will delete in my next PR.

Reply:
oops 100%. Missed this one